### PR TITLE
[BBCSPA-1307] UneditableTextField component

### PR DESCRIPTION
**Pull request by PerrineBoissieres**
Small component that allows to render text like it is a Textfield. It can also be a link.

### CODE REVIEW

**Comment by PerrineBoissieres** [comment]
Changelog added and PR rebased ;)

File: src/fakeTextField/index.unit.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+describe('FakeTextField', () => {
+  it('Should have the proper default props', () => {
+    const wrapper = renderer.create(<FakeTextField>Hello world</FakeTextField>)
+    expect(wrapper.toJSON()).toMatchSnapshot()
+  })
+
+  it('Should have the text ellipsed', () => {
+    const wrapper = renderer.create((
+      <FakeTextField ellipsis>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```
[the remark this answers is not in the document]
Comment (PerrineBoissieres) [reply src/fakeTextField/index.unit.tsx]: I used the two existing examples we have (EditMySearch dispatch step and SearchResults page) to spec it.

File: src/fakeTextField/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+export interface FakeTextFieldProps {
+  readonly children: JSX.Element|string,
+  readonly className?: Classcat.Class,
+  readonly addOn?: JSX.Element,
+  readonly href?: JSX.Element|string,
+  readonly ellipsis?: boolean
+}
+
+export const FakeTextField = ({
```
[the remark this answers is not in the document]
Comment (PerrineBoissieres) [reply src/fakeTextField/index.tsx]: Any suggestion? It's not in the design system, so it's nameless.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
-#Unreleased
+# Unreleased
+- NEW FakeTextField component
```
Comment (wakooka) [inline CHANGELOG.md]: Update name

**Pull request merged**